test/e2e/lib/install.bash

```diff
@@ -46,11 +46,34 @@ function uninstall_flux_with_helm() {
   kubectl delete crd helmreleases.flux.weave.works > /dev/null 2>&1
 }
 
+fluxctl_install_cmd="fluxctl install --namespace "${FLUX_NAMESPACE}" --git-url=ssh://git@gitsrv/git-server/repos/cluster.git --git-email=foo"
+
+function install_flux_with_fluxctl() {
+  local eol=$'\n'
+  # Use the local Flux image instead of the latest release, use a poll interval of 10s
+  # (to make tests quicker) and disable registry polling (to avoid overloading kind)
+  $fluxctl_install_cmd | \
+    sed 's%docker\.io/fluxcd/flux:.*%fluxcd/flux:latest%' | \
+    sed "s%--git-email=foo%--git-email=foo\\$eol        - --git-poll-interval=10s%" | \
+    sed "s%--git-email=foo%--git-email=foo\\$eol        - --sync-interval=10s%" | \
+    sed "s%--git-email=foo%--git-email=foo\\$eol        - --registry-exclude-image=\*%" | \
+    kubectl apply -f -
+  kubectl -n "${FLUX_NAMESPACE}" rollout status deployment/flux
+  # Add the known hosts file manually (it's much easier than editing the manifests to add a volume)
+  local flux_podname=$(kubectl get pod -n flux-e2e -l name=flux -o jsonpath="{['items'][0].metadata.name}")
+  kubectl exec -n "${FLUX_NAMESPACE}" "${flux_podname}" -- sh -c "echo '$(cat ${FIXTURES_DIR}/known_hosts)' > /root/.ssh/known_hosts"
```

Comment:
This is pretty horrible. I'm open for suggestions.

Reply:
this comment is also aimed at the sed expressions above (I tried to make a comment on the full block of code but it seems I failed)

Reply:
Doesn't the `install` package already support `additionalArguments`? What about instead of doing the `sed`, we make a small adjustment to `fluxctl install` to accept a string slice of additional arguments, and then utilize this here?

Reply:
I like your suggestions. I initially incorporated `--extra-flux-arguments`, but @squaremo thought it was better to edit the output.

@squaremo I still think there is a valid usecase here, could you reevaluate?

Reply:
@hiddeco regardless, I will _fix_ this in a separate PR. Are you OK with the rest?

Reply:
Is it possible to run the output of `fluxctl install` through kustomize to do the patching you need?